### DISCUSSION TITLE
Status of query parameters support?

### DISCUSSION THREAD

**Discussion by elmart-devo** [memgraph/memgraph#687] [Q&A] [answered]
Hi and thx for this very interesting project.

I seem to understand from the docs (https://memgraph.com/docs/cypher-manual/other-features#parameters) that parameters containing property maps are supported when appearing as literals, but not in match or merge clauses.

Just wanted to make sure 1) that I have understood correctly and 2) that situation has not changed recently and docs are outdated on that.

Thx very much.

**Comment by antejavor** [comment, marked as answer]
Hi @elmart-devo, sorry for a bit late reply. Somehow this Github discussion went under our radar. 

Yes, property maps are not supported in MATCH/MEREGE clauses, but you can use them on CREATE clause at the moment. So docs are up to date.  Is this something critical for you?

**Reply by elmart-devo** [reply]
No, it's not critical to me. At some point I though it was going to be, but it turned out not to be the case.
Thx for your reply.

**Comment by gitbuda** [comment]
A quick note from my side 😄

Our community has enthusiastically embraced [Discord](https://discord.gg/memgraph) and [Stackoverflow](https://stackoverflow.com/questions/tagged/memgraphdb) as the favored channels for sharing with our team and asking questions. We’ve decided to disable the discussions tab here on GitHub in 2 weeks.

We look forward to seeing you there!
